community/github/continuous-integration.md

```diff
@@ -99,6 +98,73 @@ All of our website repositories have several CI builds, including:
+
+A screenshot from a GitHub PR where all checks have passed.
+```
+
+Once you click on the CI output Details, you'll be taken to the `build-book` jobs screen, which looks like this:
+
+```{figure} /images/github-images/htmlproofer-build-book.png
+:alt: A screenshot of the build-book jobs page in GitHub. There's a white column to the left with the job name and run details, and a large black box to the right with all of the various dropdowns within the build-book job. The htmlproofer dropdown is just over halfway down the page.
+
+A screenshot of the build-book jobs page in GitHub.
+```
+
+When you expand the htmlproofer line, which reads Check HTML using htmlproofer, you will see the following:
+
+```{figure} /images/github-images/htmlproofer-build-book.png
+:alt: A screenshot of the build-book jobs page in GitHub with htmlproofer expanded. There's a white column to the left with the job name and run details, and a large black box to the right with all of the various dropdowns within the build-book job. The htmlproofer dropdown is just over halfway down the page and expanded to reveal information such as the run details, which checks were run, how many links were checked, whether or not HTMLProofer was successful, and how long the run took.
+
+A screenshot of the build-book jobs page in GitHub with htmlproofer expanded.
+```
+
+If all of the HTMLProofer checks pass, you won't see any errors. Instead, you will see a bright green checkmark. However, sometimes you may encounter an error. 
+Below is an image the shows a CI error associated with the build:
+
+```{figure} /images/github-images/htmlproofer-fail.png
+:alt: A screenshot of the GitHub CI notifications for a sample pyOpenSci repository. There is a header that reads All checks have passed, with the text 3 failing checks beneath it. To the right side of the header is linked text reading Hide all checks. Below the header are three rows of information. The first row reads Build Push github pages / build-book (pull_request), the second reads ci/circleci: build_book -- Your tests failed on CircleCI!, and the third reads pre-commi.ci - pr -- checks completed with failures. There is a red check next to each line of text. Beneath these rows is the text This branch has no conflicts with the base branch. Merging can be performed automatically. There is also a green Merge pull request button.
+
+A screenshot from a GitHub PR where all checks have passed.
+```
+
+When there is an error in the pyOpenSci CI build, there are a few things to consider. 
+
+1. Are the CI errors are unrelated to your pull request? pyOpenSci's CI builds run on all of the files in the repository by default. In some cases, a link might become broken simply because of the website being down. And that break has nothing to do with your pull request! In that case pyOpenSci repository maintianers can still merge the pull request, even if there are failed checks. 
```

Comment:
```suggestion
1. Are the CI errors unrelated to your pull request? pyOpenSci's CI builds run on all the files in the repository by default. Sometimes, a link might become broken simply because the website is down. And that break has nothing to do with your pull request! In that case, pyOpenSci repository maintainers can still merge the pull request, even if there are failed checks. 
```